packages/cbioportal-clinical-timeline/src/Timeline.tsx

```diff
@@ -168,12 +168,29 @@ const hoverCallback = (
                      #${uniqueId} .tl-timeline .tl-track:nth-child(${trackIndex +
                         1}) .tl-track-highlight {
                         opacity: 1 !important;
-                    }
+                     }
+                     
+                     #${uniqueId} .tl-tracklegend:nth-of-type(${trackIndex +
+                        1}) {
+                        display:block !important;
+                     }
+                     
                 `);
                 }
                 break;
             default:
-                jQuery(styleTag.current!).empty();
+                // when mouse is moving INTO the tooltip, we do not want to abide the
+                // mouseleave event. we treat it as if it's part of the track element
+                if (
+                    e.type === 'mouseleave' &&
+                    (e?.relatedTarget as Element).getAttribute('class') ===
```

Comment:
why do we need `e?` here but not `e?.type` on the line above?

Reply:
because it will definately have a type but not necessarily a relatedTarget.